README.md

```diff
@@ -27,21 +27,33 @@ To run the tests for the service, from the command line run:
 
 - Create a customer: 
     * `POST /customers`
-    * The JSON request body for creating a customer is expected to contain the following keys: ``first_name, last_name, user_name, addresses, password``
+    * The JSON request body for creating a customer is expected to contain the following keys: ``first_name, last_name, user_name, addresses, password, state,       * city, zipcode, country, and locked``
     * The expected types of the values for each of the expected keys are as follows:
         - first_name: string
-        - last_name: string
+        - last_name: string 
         - user_name: string
-        - addresses: array of string
         - password: string 
+        - address_id : integer
+        - address_line : string
+        - state : string
+        - city : string
+        - zipcode : integer
+        - country : string
+        - locked : boolean
     * Sample JSON request body format:
     ```
     {
         "first_name": "John",  
         "last_name": "Smith",  
         "username": "XXX",  
-        "addresses": ["123 Main St, Buffalo, NY", "50 John Street, NY"],  
-        "password": "XXX"  
+        "password": "XXX" 
+        "address_id": 4,      
```

Comment:
Locked is not part of the request body, and the address parameters are passed in through an address JSON object. I think it's designed to accept something like this: 

{
        "first_name": "John",  
        "last_name": "Smith",  
        "username": "XXX",  
        "password": "XXX"  
        "address": {
             "street_address": "123 Fake Road",  
             "state": "NY",
             "city": "New York",
            "zipcode": 10001,
            "country": "USA"
      }
}

I think all of these request bodies need to be checked using our Flask app w/ Postman to make sure the readme is correct